stable/kube-state-metrics/templates/_helpers.tpl

```diff
@@ -28,7 +28,7 @@ If release name contains chart name it will be used as a full name.
 Create the name of the service account to use
 */}}
 {{- define "kube-state-metrics.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create -}}
+{{- if and .Values.rbac.create .Values.serviceAccount.create -}}
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for the helper template.

Reply:
@davidkarlsen I just moved this condition here since before it was in the deployment.yaml file and was causing the issue described above. I can remove this condition from here as well if you think it is not necessary and won't cause any issues for others.